### PR TITLE
chore: use M_PI instead of PI parameter

### PR DESCRIPTION
**Pull request by Divyansh-jain2**
#### Checklist

- [x] Added description of change
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Review by realstealthninja — APPROVED** [review]
Thank you!

**Comment by realstealthninja** [comment]
have you checked all the tests?

**Comment by realstealthninja** [comment]
might need to add this before cmath 
```cpp
#define _USE_MATH_DEFINES
#include <cmath>
```

File: physics/ground_to_ground_projectile_motion
Comment (realstealthninja) [inline physics/ground_to_ground_projectile_motion:1]: What is this file

**Review by realstealthninja — CHANGES_REQUESTED** [review]

File: physics/ground_to_ground_projectile_motion.cpp [hunk 1]
```diff
@@ -30,7 +31,7 @@ namespace ground_to_ground_projectile_motion {
  * @param PI The definition of the constant PI
  * @returns Angle in degrees
  */
-double degrees_to_radians(double radian, double PI = 3.14) {
+double degrees_to_radians(double radian, double PI = M_PI) {
```
Comment (realstealthninja) [inline physics/ground_to_ground_projectile_motion.cpp:34]: Also isn't there a function in cmath which does this for you?

Comment (Divyansh-jain2) [reply physics/ground_to_ground_projectile_motion.cpp]: there is a function std::radians(degrees) but it is available only in c++20 version

File: physics/ground_to_ground_projectile_motion.cpp [hunk 2]
```diff
@@ -10,6 +10,7 @@
  */
 
 #include <cassert>   /// for assert()
+#define _USE_MATH_DEFINES
 #include <cmath>     /// for std::pow(), std::sin(), and std::cos()
```
Comment (realstealthninja) [inline physics/ground_to_ground_projectile_motion.cpp:14]: Add M_PI here as a comment

**Review by realstealthninja — APPROVED** [review]

File: physics/ground_to_ground_projectile_motion.cpp [hunk 3]
```diff
@@ -30,8 +31,10 @@ namespace ground_to_ground_projectile_motion {
  * @param PI The definition of the constant PI
```
Comment (realstealthninja) [inline physics/ground_to_ground_projectile_motion.cpp:31]:
```suggestion
```
Should remove this inturn

**Review by realstealthninja — CHANGES_REQUESTED** [review]

File: physics/ground_to_ground_projectile_motion.cpp [hunk 4]
```diff
@@ -27,11 +28,12 @@ namespace ground_to_ground_projectile_motion {
 /**
  * @brief Convert radians to degrees
  * @param radian Angle in radians
- * @param PI The definition of the constant PI
  * @returns Angle in degrees
  */
-double degrees_to_radians(double radian, double PI = 3.14) {
-    return (radian * (PI / 180));
+
+double degree_to_radians(double degree){
```
Comment (realstealthninja) [inline physics/ground_to_ground_projectile_motion.cpp:34]:
```suggestion
double degrees_to_radians(double degree){
```

**Review by realstealthninja — APPROVED** [review]

**Comment by realstealthninja** [comment]
@Divyansh-jain2 there is just a single conversation to resolve to get this merged!

**Pull request merged**